packages/dm-core-plugins/src/form/fields/ArrayField.tsx

```diff
@@ -45,7 +26,7 @@ export default function Fields(props: any) {
   })
 
   const handleAddObject = () => {
-    const name: string = `${namePath}-${fields.length}`
+    // const name: string = `${namePath}-${fields.length}`
```

Comment:
can comment be removed?

Reply:
Yeah! I just didn't know what it was for and it showed as a warning so I commented it out, but we might as well delete it since it was unused 🙂